include/deal.II/base/function_parser.h

```diff
@@ -259,11 +270,6 @@ class FunctionParser : public AutoDerivativeFunction<dim>
    */
   FunctionParser(FunctionParser &&) = delete;
 
-  /**
-   * Destructor.
-   */
-  virtual ~FunctionParser() override;
```

Comment:
Why do you no longer need this?

Reply:
We can get rid of this since we are now using PIMPL: the default destructor, generated in the header, doesn't need to call `~mu::Parser()` but instead calls `~internal::muParserBase()` which is virtual (and overridden in the source file).

Reply:
Ah, never mind. It's because you now store a pointer to the base class which is visible in the header file.